### DISCUSSION TITLE
CriticalStrech Damage with NOSPD model

### DISCUSSION THREAD

**Discussion by ppandit95** [idaholab/moose#19991] [Q&A General] [answered]
Dear MOOSE Community

I wish to replicate a simulation done in Bond Based formulation using a NOSPD formulation that involves damage so I was wondering if it is possible to use critical stretch damage criteria with the NOSPD model although I understand that generally energy or stress-based criteria is used but since I am carrying out brittle fracture I just wanted to know if it is possible or I should go with the development of energy-based damage rule.

Many Thanks
Pushkar

**Comment by GiudGiud** [comment]
@laagesen @jiangwen84 any ideas

**Reply by laagesen** [reply]
Afraid I can't help here, sorry...

**Reply by jiangwen84** [reply]
@hchen139

**Comment by hchen139** [comment, marked as answer]
It is not a good option to use bond breakage criterion in NOSPD models. A damage model will work better.

**Reply by ppandit95** [reply]
Thanks @hchen139 for elaborating on that as in one of your published paper on BA NOSPD , a critical stretch damage model was used but it seems to give inaccurate results with NOSPD model as you suggested.
